image/Dockerfile.engine

```diff
@@ -31,19 +31,22 @@ ARG BUILDTIME
 ARG PLATFORM
 ARG PRODUCT
 ARG DEFAULT_PRODUCT_LICENSE
+ARG TINI_COMMIT
```

Comment:
Do we need to pass in that argument somewhere with `--build-arg TINI_COMMIT` ?

Reply:
Hm.. good one; let me check if it's passed somewhere

Reply:
even wondering how this worked at all, because the `binaries-commits` file was removed in https://github.com/moby/moby/commit/b529d1b0936b90ae14d584c73f7332919f8d76b7